automation/run-tests.sh

```diff
@@ -189,8 +193,8 @@ while true; do
         DOCKER_IMAGE="nmstate/centos7-nmstate-dev"
         ;;
     --help)
-        echo -n "$0 [--copr=...] [--pytest-args=...] [--help] [--debug-shell] "
-        echo -n "[--el7] "
+        echo -n "$0 [--copr=...] [--customize=...] [--debug-shell] [--el7]"
```

Comment:
nit: I think the order should be per importance and usage, placing `copr` and this new one first is not that useful/helpful for a user.

Reply:
I guess or preferences differ. For me a sorted list makes it easier to find an argument that I vaguely remember or assume.

Reply:
Actually the order reflects the current order of importance for me, too ;-P - If we want to overengineer the script we need to make `--test-type` an argument, since it is not really optional (except for the default of all), so it would be `run-tests.sh integ`. Not really in the mood to do this now, though. I added a new commit to make the help a least readable by disabling xtrace for it.